index.js

```diff
@@ -25,66 +25,90 @@ function getRandomNumber(min, max) {
 function checkGuess() {
   // Get value from guess input element
   const guess = parseInt(guessInput.value, 10);
-  attempts = attempts + 1;
+  //attempts = attempts + 1;
 
   hideAllMessages();
 
+  if (guess <= 0) {
```

Comment:
Should also check for integer here.  Currently accepts floating point numbers.

Reply:
As I understand it need to be like this: if (guess <= 0 && Number.isInteger(guess)) {
Thanks a Lot!

Reply:
guess should be > 0, < 100 and number.isInteger() must be true.